File: ClientApp/src/app/core/interfaces/Product/Response/option-cart-response.ts
import {MultipleImagesResponse} from "./multipleImagesResponse";

export interface OptionCartResponse {
  title: string;
  image: MultipleImagesResponse;
  price: number;
  offerPrice: number;
  discountValue: number;
  quantityAvailable: number;
  weight: number;
}

File: Contracts/Model/Product/Response/OptionCartResponse.cs
namespace Contracts.Model.Product.Response;
public class OptionCartResponse
{
	public string Title { get; set; }
	public string Description { get; set; }
	public MultipleImagesResponse Image { get; set; }
	public double Price { get; set; }
	public int QuantityAvailable { get; set; }
}
